library/src/methods/brand/brand.ts

```diff
@@ -633,6 +672,20 @@ export function brand<
 /**
  * Brands the output type of a schema.
  *
+ * @example
```

Comment:
To keep the source code as simple as possible, I prefer not to add examples. I will add examples in the future to the API reference on the website.